ui/library/src/context/ScrollContext.ts

```diff
@@ -172,8 +172,15 @@ export function useScrollContextProps(): IScrollContext {
   }, []);
 
   const getMaxVisibleElement = (visibleElements: Map<any, number>): any => {
```

Comment:
Better replacing `any` with specific types. (I believe they should be `number` from your code)

Reply:
You may want to change the function name as well, since it's finding maximal page number instead of maximal visible element.

Reply:
@yensung my bad this is supposed to be getting rid of since i have a util file for it. But also the reason for `getMaxVisibleElement` is it can be used for outline too